src/braket/experimental/autoqasm/program/program.py

```diff
@@ -87,6 +88,65 @@ class ProgramMode(Enum):
     """For program conversion inside a context where only pulse operations are allowed."""
 
 
+class MainProgram(SerializableProgram):
+    """Represents an AutoQASM program that has not yet been generated. The program can be built by
```

Comment:
"not yet been generated" is a little vague.

```suggestion
    """Represents an AutoQASM program. The program can be built by
```